ecdsa/src/gadgets/ecdsa.rs

```diff
@@ -22,48 +24,48 @@ pub struct ECDSASignatureTarget<C: Curve> {
     pub s: NonNativeTarget<C::ScalarField>,
 }
 
-impl<F: RichField + Extendable<D>, const D: usize> CircuitBuilder<F, D> {
-    pub fn verify_message(
-        &mut self,
-        msg: NonNativeTarget<Secp256K1Scalar>,
-        sig: ECDSASignatureTarget<Secp256K1>,
-        pk: ECDSAPublicKeyTarget<Secp256K1>,
-    ) {
-        let ECDSASignatureTarget { r, s } = sig;
-
-        self.curve_assert_valid(&pk.0);
-
-        let c = self.inv_nonnative(&s);
-        let u1 = self.mul_nonnative(&msg, &c);
-        let u2 = self.mul_nonnative(&r, &c);
-
-        let point1 = self.fixed_base_curve_mul(Secp256K1::GENERATOR_AFFINE, &u1);
-        let point2 = self.glv_mul(&pk.0, &u2);
-        let point = self.curve_add(&point1, &point2);
-
-        let x = NonNativeTarget::<Secp256K1Scalar> {
-            value: point.x.value,
-            _phantom: PhantomData,
-        };
-        self.connect_nonnative(&r, &x);
-    }
+pub fn verify_message_circuit<F: RichField + Extendable<D>, const D: usize>(
```

Comment:
Just curious, is this because we're expecting ECDSA verification to be done by a dedicated circuit?